Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -12,11 +12,21 @@ private struct BackForwardViewUX {
     static let RowHeight: CGFloat = 50
 }
 
+/// Provides information about the size of various BrowserViewController's subviews.
+protocol BrowserViewControllerFrameInfoProvider: AnyObject {
```

Comment:
Proposition so the name is a little bit shorter
```suggestion
protocol BrowserFrameInfoProvider: AnyObject {
```

Reply:
Yes sounds good